### DISCUSSION TITLE
OpenFilePicker does not work in platforms other than windows

### DISCUSSION THREAD

**Discussion by garawaa** [AvaloniaUI/Avalonia#9758] [Q&A]
I tried the some code for android and web platforms. But does not work. For example:

```
        var openfiledialog = new OpenFileDialog();
        var result = await openfiledialog.ShowAsync(new Window());
```

**Comment by timunie** [comment]
Hi @garawaa , 

which version of Avalonia do you use? WASM support and Android support is very very much improved in upcomming v11.0. You're welcome to test our the previews. There we have a StorageProvider which may help you. Anyway, Browser is a sandbox, so you may not be able to access everything.
